src/modules/remote.json

```diff
@@ -0,0 +1,178 @@
+          "params": [],
+          "result": {
+            "name": "Remote List example",
+            "value": {
+              "list": [
+                {
+                  "model": "PR2",
+                  "batteryPercentage": 70
+                }
+              ]
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "startPairing",
+      "summary": "Setup the device to allow Remote control to be paired on the next keypress.",
+      "params": [],
+      "result": {
+        "name": "startPairingResult",
+        "summary": "The state of currently paired remote",
+        "schema": {
+          "$ref": "#/components/schemas/StartPairingResult"
+        }
+      },
+      "examples": [
+        {
+          "name": "Remote State Examples",
+          "params": [],
+          "result": {
+            "name": "Remote State Example",
+            "value": {
+              "startPairingResult": true
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "onRemotePairing",
+      "tags": [
+        {
+          "name": "event"
+        }
+      ],
+      "summary": "Listen for events when the remote control is pairing.",
+      "params": [
+        {
+          "name": "listen",
+          "required": true,
+          "summary": "whether to listen, or stop listening",
+          "schema": {
+            "type": "boolean"
+          }
+        }
+      ],
+      "result": {
+        "name": "pairingResult",
+        "summary": "The state of currently paired remote",
+        "schema": {
+          "oneOf": [
+            {
+              "$ref": "https://meta.comcast.com/firebolt/types#/definitions/ListenResponse"
+            },
+            {
+              "$ref": "#/components/schemas/RemotePairResult"
+            }
+          ]
+        }
+      },
+      "examples": [
+        {
+          "name": "Default Example",
+          "params": [
+            {
+              "name": "listen",
+              "value": true
+            }
+          ],
+          "result": {
+            "name": "event",
+            "value": {
+              "pairingResult": true
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "components": {
+    "schemas": {
+      "RemoteControlState": {
+        "title": "RemoteList",
+        "type": "object",
+        "description": "Contains a list of remotes paired to the device.",
+        "minProperties": 1,
+        "additionalProperties": false,
+        "required": ["list"],
+        "properties": {
+          "list": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/RemoteState"
+            }
+          }
+        }
+      },
+      "RemoteState": {
+        "title": "RemoteState",
+        "type": "object",
+        "description": "The type of keyboard to show to the user",
+        "minProperties": 2,
+        "additionalProperties": false,
+        "required": ["model", "batteryPercentage"],
```

Comment:
I think if it has a default then it is not required, so maybe you shouldn't have both